llvm/include/llvm/IR/Attributes.td

```diff
@@ -218,8 +218,12 @@ def ReadNone : EnumAttr<"readnone", [ParamAttr]>;
 /// Function only reads from memory.
 def ReadOnly : EnumAttr<"readonly", [ParamAttr]>;
 
+/// Function is marked to be analyzed by the realtime sanitizer.
 def Realtime : EnumAttr<"realtime", [FnAttr]>;
 
+/// Function can be called from realtime functions and is omitted from realtime sanitizer analysis.
+def RealtimeBypass : EnumAttr<"realtime_bypass", [FnAttr]>;
```

Comment:
Do we like this name as "Realtime Bypass"? What other name should we use if not?

Reply:
As discussed on Discord, we like `NoSanitizeRealtime`, to match the existing `NoSanitizeBounds` and `NoSanitizeCoverage`